script/upgradedb

```diff
@@ -104,7 +101,4 @@ if ($prepare_upgrades) {
     $dh->prepare_upgrade({from_version => $prev_version, to_version => $version});
 }
 
-if ($upgrade_database) {
-    OpenQA::Schema::deployment_check($schema, $force);
-}
-
+OpenQA::Schema::deployment($schema, $force) if $upgrade_database;
```

Comment:
This (and the other occurrences) can be written as `$schema->deploy($force)` which would look much nicer. So if you want to change the style I'd change it to that.

Reply:
I have done that but in a separate commit on top as the method was not really looking like a method yet.